src/api/MoneyMoovApiClient.ts

```diff
@@ -0,0 +1,73 @@
+import { deletePaymentRequest, getPaymentRequests, voidPaymentRequest } from "./PaymentRequestClient";
+import { PaymentRequestPageResponse } from "./types/ApiResponses";
+import { ApiError } from "./types/ApiResponses";
+
+class MoneyMoovApiClient {
+  apiUrl: string;
+  token: string;
+
+  /**
+   * @param apiUrl The api base url
+   * @param token The token

```

Comment:
Can we specify a bit more what this token is, and maybe where to get it too?

I believe this will be very beneficial for people using the library, as we all know, there's loads of people who don't exactly read docs, so might be good to have it as IntelliSense here too.